common.js

```diff
@@ -1943,6 +1943,237 @@ const allSoloCars = {
       '2012': ['gs']
     },
   },
+  'Subaru': {
+    'BRZ (non-Performance Package)': {
+      '2013': ['ds', 'stx', 'csp', 'dsp', 'ssc'],
+      '2014': ['ds', 'stx', 'csp', 'dsp', 'ssc'],
+      '2015': ['ds', 'ssc', 'stx', 'dsp'],
+      '2016': ['ds', 'ssc', 'stx', 'dsp'],
+      '2017': ['ds', 'stx', 'dsp'],
+      '2018': ['ds', 'cs', 'stx', 'dsp'],
+      '2019': ['ds', 'stx'],
+      '2020': ['stx'],
+    },
+    'BRZ (with Performance Package)': {
+      '2013': ['stx', 'csp', 'dsp', 'ssc'],
```

Comment:
BRZ performance package was only offered in 2017 and up, everything else looks good.